Source/Parallelization/WarpXComm_K.H

```diff
@@ -522,6 +522,13 @@ void warpx_interp (const int j,
 {
     using namespace amrex;
 
+    // Pad input array with zeros beyond ghost cells
+    // for out-of-bound accesses due to large-stencil operations
+    const auto arr_safe = [arr_fine] (const int jj, const int kk, const int ll) noexcept
+    {
+        return arr_fine.contains(jj,kk,ll) ? arr_fine(jj,kk,ll) : 0.0_rt;
+    };
+
```

Comment:
@WeiqunZhang confirmed that this device code for padding should be reasonably fast.